website/docs/secrets/manage-secrets-gui.mdx

```diff
@@ -3,18 +3,131 @@ title: Manage Secrets GUI
+
+- **Secret path:** This is the path to the external secret within the cluster.
+
+- **Property:** This is the property or key that is associated with the external secret.
+
+- **Version:** This shows the version of the external secret, which may be blank if no version has been specified.
+
+Understanding the information provided on the External Secret Details page can help you to manage and troubleshoot your external secrets as needed.
+
+
+### Understanding Events
+
+![External Secret Events](./imgs/external-secret-events-1.png)
+
+The following events can be expected when using the Manage Secrets GUI to manage external secrets:
+
+- **Updated:** This event indicates that an existing external secret has been successfully updated with new data.
+
+- **Not Ready:** This event indicates that there was an issue with the secret store when trying to access or synchronize external secrets. This includes situations such as the secret store being unavailable or not ready to handle requests, or issues with authentication when trying to access the secret store.
+
+Understanding these events can help you to troubleshoot issues that may arise when managing external secrets using the Manage Secrets GUI. In particular, if you encounter a "Not Ready" event, you may need to check your secret store credentials and ensure that the secret store is operational before proceeding with any further actions.
+
+### Sync External Secrets
+
+Once you have created an External Secret, it is important to keep it up to date with the latest changes in the corresponding external secret manager. This can be achieved by syncing the External Secret with its source using the Sync feature provided by WeaveWorks Enterprise.
+
+To sync an External Secret, navigate to the List External Secrets page and select the External Secret you want to sync. Once you are on the External Secret Details page, click on the Sync button on the top left corner of the page. This will trigger a sync operation for the selected External Secret.
+
+It is important to note that syncing an External Secret triggers a refresh operation for the associated K8s Secret as well. This means that any updates made to the external secret manager will be immediately reflected in the K8s Secret associated with the External Secret, without waiting for the refresh interval.
+
+## SOPS
+
+### Getting Started with SOPS
+
+[SOPS](https://github.com/mozilla/sops#usage) provides an encryption mechanism for YAML files that are stored in Git repositories. SOPS allows users to encrypt secrets within YAML files, and decrypt them when they are needed.
+
+
+### Create SOPS Secret
+
+
+### Prerequisites
+
+To follow this guide you’ll need a kustomization object contains your public key. Please see the [get started guide](//url-for-create-kustomize).
+
+To create a new SOPS secret, start by clicking on the "Create new sops secret" button.
```

Comment:
You dont have any new headers which means that everything is positioned under the `Prerequisites` header.